src/components/Button/Button.css

```
@@ -585,9 +585,9 @@
  *
  * ====================
  *
- * @depracated Пользователям следует использовать `ButtonGroup` (см. пример `RichCell/Readme.md`).
+ * @deprecated Пользователям следует использовать `ButtonGroup` (см. пример `RichCell/Readme.md`).
  *
- * TODO Сейчас стиль сохранён для обртаной совместимости. Удалить в следующей мажорной версии.
+ * TODO Сейчас стиль сохранён для обратной совместимости. Удалить в следующей мажорной версии.
```

```suggestion
 * TODO Сейчас стиль сохранён для обратной совместимости. Удалить в 5.0.0.
```
Раз уж влезли сюда, предлагаю поменять еще и вот эту штуку, чтобы можно было грепнуть потом всё по `5.0.0`. 😉